softbinding-algorithm-entry-schema.json

```
@@ -1,110 +1,114 @@
 {
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "type": "object",
-    "properties": {
-        "identifier": {
-            "type": "integer",
-            "minimum": 0,
-            "maximum": 65535,
-            "description": "This identifier will be assigned when the soft binding algorithm is added to the list."
-        },
-        "deprecated": {
-            "type": "boolean",
-            "default": false,
-            "description": "Indicates whether this soft binding algorithm is deprecated. Deprecated algorithms shall not be used for creating soft bindings.  Deprecated algorithms may be used for resolving soft bindings but this behaviour is discouraged."
-        },
-        "alg": {
-            "type": "string",
-            "pattern": "(c2pa\\.|[A-Za-z0-9\\-\\.]+)",
-            "description": "Entity-specific namespace as specified for C2PA Assertions labels that shall begin with the Internet domain name for the entity similar to how Java packages are defined (e.g., `com.example.algo1`, `net.example.algos.algo2`)"
-        },
-        "type": {
-            "type": "string",
-            "enum": [
-                "watermark",
-                "fingerprint"
-            ],
-            "description": "Type of soft binding implemented by this algorithm."
-        },
-        "decodedMediaTypes": {
-            "type": "array",
-            "minItems": 1,
-            "items": {
+    "$id": "https://c2pa.org/schemas/softbinding-algorithm-list.json",
```

very good point, we were missing the ID indeed! Should this not point to the actual schema instead?
So:
```suggestion
    "$id": "https://spec.c2pa.org/specifications/specifications/2.2/specs/C2PA_Specification.html#_soft_binding_algorithm_list",
```

---

It doesn't matter.  It used to be that `$id` needed to resolve to something, but nowadays it's just an identifier.  We can set it to whatever.